### PR TITLE
Definite titular titles

### PR DESCRIPTION
**Pull request by ElMariuso**
## Changelog:
- Adapted all titular titles by giving an adjective and an article
- Sort this files: wc_other_titles_l_english.yml, wc_holy_order.txt, wc_titular_titles.txt, wc_religious_titles.txt
- Added new english localization
- Added french localization

# How to test:
- Open the game
- Check the titular titles, the holy orders and the religious titles

#  Check if:
- Unused titles from wc_other_titles_l_english.yml are really unused.
- If the new titles names look good. I added suffixes to many of them (for example: Blackrock clan rather than just Blackrock). We have to see if it looks good.

### CODE REVIEW

File: localization/french/wc_other_titles_l_french.yml [hunk 1]
```diff
@@ -1,453 +1,802 @@
...
+ d_dragonmaw:0 "Clan Gueule-de-dragon"
+ d_dragonmaw_adj:0 "gueule-de-dragon"
+ d_dragonmaw_article:0 "le "
+
+ d_warsong:0 "Clan Chanteguerre"
+ d_warsong_adj:0 "chanteguerre"
+ d_warsong_article:0 "le "
+ 
+ d_shattered_hand:0 "Clan de la Main-Brisée"
+ d_shattered_hand_adj:0 "main-Brisée"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:43]: majuscule à retirer

File: localization/french/wc_other_titles_l_french.yml [hunk 2]
```diff
@@ -1,453 +1,802 @@
...
+ d_defias_brotherhood:0 "Confrérie défias"
+ d_defias_brotherhood_adj:0 "défias"
+ d_defias_brotherhood_article:0 "la "
+
+ d_syndicate:0 "Syndicat"
+ d_syndicate_adj:0 "du Syndicat"
+ d_syndicate_article:0 "le "
+ 
+ d_karazhan:0 "Karazhan"
+ d_karazhan_adj:0 "Karazhanienne"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:71]: pareil j'enlèverais la majuscule ici

File: localization/french/wc_other_titles_l_french.yml [hunk 3]
```diff
@@ -1,453 +1,802 @@
...
+ 
+ d_caer_darrow:0 "Caer Darrow"
+ d_caer_darrow_adj:0 "Caer Darrowienne"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:74]: idem

File: localization/french/wc_other_titles_l_french.yml [hunk 4]
```diff
@@ -1,453 +1,802 @@
...
+ e_black_dragonflight:0 "Vol noir"
+ e_black_dragonflight_adj:0 "obsidienne"
+ e_black_dragonflight_article:0 "le "
+
+ e_red_dragonflight:0 "Vol rouge"
+ e_red_dragonflight_adj:0 "vermillon"
+ e_red_dragonflight_article:0 "le "
+
+ e_blue_dragonflight:0 "Vol bleu"
+ e_blue_dragonflight_adj:0 "azure"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:135]: pas de "e" à azur

File: localization/french/wc_other_titles_l_french.yml [hunk 5]
```diff
@@ -1,453 +1,802 @@
...
+ k_dampwick_cartel_adj:0 "mèchumide"
+ k_dampwick_cartel_article:0 "le "
+
+ ### Pirates ###
+ d_blackwater_raiders:0 "Écumeurs des Flots noirs"
+ d_blackwater_raiders_adj:0 "Flots noirs"
+ d_blackwater_raiders_article:0 "les "
+
+ d_bloodsail_buccaneers:0 "Boucaniers de la Voile sanglante"
+ d_bloodsail_buccaneers_adj:0 "Voile sanglante"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:173]: sans maj

File: localization/french/wc_other_titles_l_french.yml [hunk 6]
```diff
@@ -1,453 +1,802 @@
...
+
+ ### Draenei ###
+ k_exodar:0 "Exodar"
+ k_exodar_adj:0 "exodarienne"
+ k_exodar_article:0 "l'"
+
+ ### Nagas ###
+ d_coilfang:0 "Tribu Glisseroc"
+ d_coilfang_adj:0 "glisseroc"
+ d_coilfang_article:0 "the "
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:366]: oubli de traduction

File: localization/french/wc_other_titles_l_french.yml [hunk 7]
```diff
@@ -1,453 +1,802 @@
...
- e_pantheon:0 "Panthéon"
+ k_cenarion_circle_article:0 "le "
+
+ d_cenarion_circle:0 "Cercle cénarien"
+ d_cenarion_circle_adj:0 "cénarienne"
+ d_cenarion_circle_article:0 "le "
+
+ ### Pandaria ###
+ e_pandaria_empire:0 "Pandarie"
+ e_pandaria_empire_adj:0 "pandaroise"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:387]: on ne dirait pas plutôt pandarène ? c'est le cas pour l'adjectif de culture

File: localization/french/wc_other_titles_l_french.yml [hunk 8]
```diff
@@ -1,453 +1,802 @@
...
+ d_alakir_worship_adj:0 "$ALAKIR_POSSESSIVE$"
+ d_alakir_worship_article:0 "le "
+
+ ### Light group ###
+ d_church_of_the_holy_light:0 "Église de la Lumière"
+ d_church_of_the_holy_light_adj:0 "sacrée"
+ d_church_of_the_holy_light_article:0 "l'"
+
+ d_church_scarlet_light:0 "Monastère Écarlate"
+ d_church_scarlet_light_adj:0 "Écarlate"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:489]: majuscule à retirer

File: localization/french/wc_other_titles_l_french.yml [hunk 9]
```diff
@@ -1,453 +1,802 @@
...
+ 
+ d_guratali_council:0 "Conseil $gurubism_adherent$"
+ d_guratali_council_adj:0 "$gurubism_adherent$"
+ d_guratali_council_article:0 "le "
+ 
+ d_drakatali_council:0 "Conseil $drakkism_adherent$"
+ d_drakatali_council_adj:0 "$drakkism_adherent$"
+ d_drakatali_council_article:0 "le "
+ 
+ d_faratali_council:0 "Conseil $farrism_adherent$ Priesthood"
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:555]: tu as oublié "priesthood"

File: localization/french/wc_other_titles_l_french.yml [hunk 10]
```diff
@@ -1,453 +1,802 @@
...
- d_forsaken:0 "Réprouvés"
- d_forsaken_adj:0 "réprouvée"
- UNDERCITY:0 "Fossoyeuse"
- UNDERCITY_ADJ:0 "fossoyeuse"
+ d_tirisgarde_article:0 "la "
+ e_valarjar:0 "Salles des Valeureux"
+ d_ravenholdt:0 "Ravenholdt"
+ d_ravenholdt_adj:0 "ravenholdt"
+
+ ### Mercenaries bands ? ###
```
Comment (hamjustine) [inline localization/french/wc_other_titles_l_french.yml:646]: c'est normal qu'il manque toute cette partie ?

File: localization/french/wc_other_titles_l_french.yml [hunk 11]
```diff
@@ -1,453 +1,802 @@
...
+ #########################
+ ### Orcs ###
  e_horde:0 "Horde"
- e_horde_adj:0 "de la Horde"
+ e_horde_adj:0 "Horde"
+ e_horde_article:0 "la "
+ 
+ k_orc_blackrock:0 "Clan Rochenoire"
+ k_orc_blackrock_adj:0 "rochenoire"
+ k_orc_blackrock_article:0 "the "
```
[the remark this answers is not in the document]
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml]: done in 9b43c30b8

File: localization/french/wc_other_titles_l_french.yml [hunk 12]
```diff
@@ -1,453 +1,802 @@
...
+
+ d_stormreaver:0 "Clan Foudreguerre"
+ d_stormreaver_adj:0 "foudreguerre"
+ d_stormreaver_article:0 "le "
+
+ d_bleeding_hollow:0 "Clan de l'Orbite-Sanglante"
+ d_bleeding_hollow_adj:0 "orbite-Sanglante"
```
[the remark this answers is not in the document]
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:19]: done in c9e28f687

File: localization/french/wc_other_titles_l_french.yml [hunk 13]
```diff
@@ -1,453 +1,802 @@
...
+ d_frostwolf:0 "Clan Loup-de-givre"
+ d_frostwolf_adj:0 "loup-de-givre"
+ d_frostwolf_article:0 "le "
+
+ d_twilights_hammer:0 "Clan Marteau du crépuscule"
+ d_twilights_hammer_adj:0 "marteau du crépuscule"
+ d_twilights_hammer_article:0 "le "
+
+ d_burning_blade:0 "Clan de la Lame-Ardente"
+ d_burning_blade_adj:0 "lame-Ardente"
```
[the remark this answers is not in the document]
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:31]: done in c9e28f687

File: localization/french/wc_other_titles_l_french.yml (hunk 1 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:43]: done in c9e28f687

File: localization/french/wc_other_titles_l_french.yml (hunk 2 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml]: done in b1dc39279

File: localization/french/wc_other_titles_l_french.yml (hunk 3 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml]: done in b1dc39279

File: localization/french/wc_other_titles_l_french.yml (hunk 4 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:135]: done in 28e678154

File: localization/french/wc_other_titles_l_french.yml (hunk 5 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:173]: done in 639915e3e

File: localization/french/wc_other_titles_l_french.yml (hunk 6 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml]: done in 75c1ec1dc

File: localization/french/wc_other_titles_l_french.yml (hunk 8 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:489]: done in 235dd4ec0

File: localization/french/wc_other_titles_l_french.yml (hunk 9 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:555]: done in 48f2aa718

File: localization/french/wc_other_titles_l_french.yml (hunk 7 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:387]: Pour moi c'est l'adjectif de la race donc du coup je me suis dis que j'allais mettre autre chose.

File: localization/french/wc_other_titles_l_french.yml (hunk 10 above)
Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml:646]: Si c'est inutilisé, je ne vois pas l'intérêt de la traduire.

Comment (hamjustine) [reply localization/french/wc_other_titles_l_french.yml:646]: Ha oui, c'est mon mauvais

File: localization/french/wc_other_titles_l_french.yml (hunk 7 above)
Comment (hamjustine) [reply localization/french/wc_other_titles_l_french.yml:387]: comme dit, même une pêche prend l'adjectif pandarène, alors que c'est plutôt lié au territoire je pense plutôt qu'a la culture ou la race https://fr.wowhead.com/item=74660/p%C3%AAche-pandar%C3%A8ne

Comment (ElMariuso) [reply localization/french/wc_other_titles_l_french.yml]: Dans ce cas... Done in cde7fa852

**Review by hamjustine — APPROVED** [review]

File: localization/english/wc_other_titles_l_english.yml [hunk 14]
```diff
@@ -1,173 +1,674 @@
...
+ #########################
+ ### Orcs ###
  e_horde:0 "Horde"
  e_horde_adj:0 "Horde"
  e_horde_article:0 "the "
  
- e_empire_of_zul:0 "Empire of Zul"
- e_empire_of_zul_adj:0 "Zulian"
- e_empire_of_zul_article:0 "the "
+ k_orc_blackrock:0 "Blackrock clan"
```
Comment (zumbak04) [inline localization/english/wc_other_titles_l_english.yml:10]: All words in title name except `the` should be capitalized.

File: localization/english/wc_other_titles_l_english.yml [hunk 15]
```diff
@@ -1,173 +1,674 @@
...
+ e_blue_dragonflight_article:0 "the "
+
+ e_bronze_dragonflight:0 "Bronze Dragonflight"
+ e_bronze_dragonflight_adj:0 "Bronze"
+ e_bronze_dragonflight_article:0 "the "
+
+ ### Goblins ###
  e_trade_coalition:0 "Trade Coalition"
- d_blackwater_raiders:0 "Blackwater Raiders"
+ e_trade_coalition_adj:0 "Goblin"
```
Comment (zumbak04) [inline localization/english/wc_other_titles_l_english.yml:144]: You assume it's controlled by goblins.
```suggestion
 e_trade_coalition_adj:0 "Coalition"
```

File: localization/english/wc_other_titles_l_english.yml [hunk 16]
```diff
@@ -1,173 +1,674 @@
...
+ ### Titans ###
+ e_pantheon:0 "Pantheon"
+ 
+ #########################
+ #		Holy Orders		     #
+ #########################
+ ### Humans ###
+ d_brotherhood_of_the_horse:0 "Brotherhood of the Horse"
+
+ d_order_of_the_silver_hand:0 "Order of the Silver Hand"
```
Comment (zumbak04) [inline localization/english/wc_other_titles_l_english.yml:406]: Where is the adj, article?

File: localization/english/wc_other_titles_l_english.yml [hunk 17]
```diff
@@ -1,173 +1,674 @@
...
- d_church_scarlet_light:0 "Scarlet Monastery"
- d_church_scarlet_light_article:0 "the "
- 
- d_naaru:0 "Faithful of Naaru"
- d_hand_of_argus:0 "Hand of Argus"
- d_bloodscalp:0 "Bloodscalp"
- d_skullsplitter:0 "Skullsplitter"
- 
+ #########################
+ #		Without land		   #
```
Comment (zumbak04) [inline localization/english/wc_other_titles_l_english.yml:3]: Again, do no sort the localization files. It's said in [Our Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Localization-Guidelines).

You can move religious and holy order titles to different files. It's fine, it's allowed.

File: localization/english/wc_other_titles_l_english.yml [hunk 18]
```diff
@@ -213,10 +714,10 @@
  d_frozen_sea_fleet:0 "Frozen Sea Fleet"
  d_frozen_sea_fleet_adj:0 "Frozen Sea"
 
- d_cult_of_sunwell:0 "Sunwell Cult"
- d_cult_of_sunwell_adj:0 "Sunwell"
- d_cult_of_sunwell_article:0 "the "
+ d_gnomeregan_covert_ops:0 "Gnomeregan Covert Ops"
+ d_gnomeregan_covert_ops_adj:0 "Gnomeregan Covert"
```
Comment (zumbak04) [inline localization/english/wc_other_titles_l_english.yml:719]: Where is the article?

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/english/wc_other_titles_l_english.yml (hunk 17 above)
Comment (ElMariuso) [reply localization/english/wc_other_titles_l_english.yml]: done in 31e532701

File: localization/english/wc_other_titles_l_english.yml (hunk 16 above)
Comment (ElMariuso) [reply localization/english/wc_other_titles_l_english.yml]: fixed in 58805b8bb

File: localization/english/wc_other_titles_l_english.yml (hunk 14 above)
Comment (ElMariuso) [reply localization/english/wc_other_titles_l_english.yml]: done in 7c837aee4 and e11de47e1

File: localization/english/wc_other_titles_l_english.yml (hunk 15 above)
Comment (ElMariuso) [reply localization/english/wc_other_titles_l_english.yml]: done in ee5060e26

File: localization/english/wc_other_titles_l_english.yml (hunk 18 above)
Comment (ElMariuso) [reply localization/english/wc_other_titles_l_english.yml]: done in 2b6bd7448

File: common/landed_titles/wc_religious_titles.txt [hunk 19]
```diff
@@ -410,8 +415,7 @@ d_tidemother = {
 	# capital = 1796 # Lesturrom
 # }
 
-### arcane_group
-# cult_of_sunwell
```
Comment (zumbak04) [inline common/landed_titles/wc_religious_titles.txt:414]: Why you deleted these comments? It was clearer before. Mentioned the exact name of the religion and the faith.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/landed_titles/wc_religious_titles.txt (hunk 19 above)
Comment (ElMariuso) [reply common/landed_titles/wc_religious_titles.txt:414]: Readded in 13f4d1c0e

Comment (zumbak04) [reply common/landed_titles/wc_religious_titles.txt:414]: > Readded in [13f4d1c](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/13f4d1c0edcdd1022fb10521173e9dc6436c8064)

That's not what I meant by the exact game. `arcane_group` and `holy_light` are exact names.

Comment (ElMariuso) [reply common/landed_titles/wc_religious_titles.txt]: back in 56fb20691

File: common/landed_titles/wc_religious_titles.txt [hunk 20]
```diff
@@ -3,11 +3,11 @@
 @always_primary_score = 1000
 
 #########################
-#		Religion Head		#
+#	Religion Head       #
 #########################
 
-###air_group
```
Comment (zumbak04) [inline common/landed_titles/wc_religious_titles.txt:9]: I mean the religion name is important too.

Comment (ElMariuso) [reply common/landed_titles/wc_religious_titles.txt:9]: readded in 67226e6e0

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**